lib/sgf/gtp_writer.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+class SGF::GtpWriter < SGF::Writer
+  attr_writer :upside_down
+
+  def initialize
+    @upside_down = false
+  end
+
+  private
+
+  def gtp_move(node)
+    pps = node.properties
+    if pps['SZ']
+      @boardsize = pps['SZ'].to_i
+      out = []
+      out << "komi #{pps['KM']}" if pps['KM']
+      out << "boardsize #{pps['SZ']}\nclear_board"
+      pps['AB'].each {|pos| out << to_play("B", pos) } if pps['AB']
```

Comment:
If you have AB, it seems reasonable to think one might also have AW, is that not so?

Reply:
well, handicaps for Whites are uncommon, but maybe for a game fork, sure I'll add just in case